### ISSUE TITLE
❌ Unknown OS. Please send the result of 'uname -o' to the author.

### ISSUE THREAD

**Issue by sam-artuso** [marcopaganini/termotp#2]
I can't install the latest version:

```
$ wget -q -O/tmp/install \
  'https://raw.githubusercontent.com/marcopaganini/installer/master/install.sh' && \
  sudo sh /tmp/install marcopaganini/termotp
Password:
❌ Unknown OS. Please send the result of 'uname -o' to the author.
```

I'm using MacOS Sonoma 14.3.

```
$ uname -o
Darwin
```

**Comment by marcopaganini** [comment]
So I don't have a Mac around to test this and may need some quick help from you. I've attached a tar file with a cross-compiled binary for MacOS. Can you try to unpack this file (tar xvf), do a `chmod 755 termotp` and then try to run it?
If this works, I'll submit the proper fix for the installation files.

[termotp-darwin-amd64.tar.gz](https://github.com/marcopaganini/termotp/files/14325044/termotp-darwin-amd64.tar.gz)

**Comment by marcopaganini** [comment]
Oh, and here's the ARM64 version of termotp if you happen to have an arm based mac:
[termotp-darwin-arm64.tar.gz](https://github.com/marcopaganini/termotp/files/14325060/termotp-darwin-arm64.tar.gz)

**Comment by sam-artuso** [comment]
Just tested them and both work fine 👍🏼 .

**Comment by marcopaganini** [comment]
Nice!
Would it also be possible to send me the output of `uname -m` ?

**Comment by sam-artuso** [comment]
`arm64`

**Comment by marcopaganini** [comment]
Thanks! This should now be fixed. Can you try that installation procedure again?

**Comment by sam-artuso** [comment]
Not working yet. I think you should add an `arm64` case [here](https://github.com/marcopaganini/installer/blob/a97f8cb158b6cf048fb3d16a65a21cbeb3bf5075/install.sh#L12).

**Comment by marcopaganini** [comment]
Great, thanks! This should be fixed by [a9436f7](https://github.com/marcopaganini/installer/commit/a9436f786720093a36132929da1285e8dbcf32a9)
